.nvmrc

```diff
@@ -1 +1 @@
-16.20.0
+18.20.0
```

Comment:
Just a note: This _should_ be fine, but there's a chance it could cause other things to break.

Reply:
Is there anything in CI land that might be impacted? eg node docker images